### DISCUSSION TITLE
What does the "Hide omnibox" option do?

### DISCUSSION THREAD

**Discussion by 9t8** [violentmonkey/violentmonkey#1526] [Q&A] [answered]
I can't figure out what it changes.

**Comment by tophf** [comment, marked as answer]
Omnibox is the address bar (in the editor window).

**Reply by 9t8** [reply]
Where does it appear?

**Reply by tophf** [reply]
In the editor window.

**Reply by 9t8** [reply]
Oh, I see it now. Maybe you can make the description clearer by changing it to something like "Hide omnibox in popup editors", or possibly remove the setting since I don't see why anyone would want the omnibar to show. Thanks for the fast response.

**Reply by tophf** [reply]
How about grouping it visually instead?
![image](https://user-images.githubusercontent.com/1310400/178935424-4955b477-c145-4cec-9e37-60c1909f86d9.png)

**Reply by 9t8** [reply]
I think that clarifies it. But, I don't think this setting matters enough to be included. If on, it wastes a small amount of screen space. If off, it hides some useless information that can still easily be obtained.

**Reply by tophf** [reply]
Some people use the omnibox to access other extensions' icons, which may contain important functionality.

**Reply by 9t8** [reply]
I hadn't thought of that. I concede my point.
